src/js/actions/AppActions.js

```diff
@@ -3,146 +3,145 @@ import app    from '../lib/app';
 
 import LibraryActions       from './LibraryActions';
 import PlaylistsActions     from './PlaylistsActions';
-import NotificationsActions from './NotificationsActions';
 import PlayerActions        from './PlayerActions';
-import QueueActions         from './QueueActions';
 import SettingsActions      from './SettingsActions';
 
 const globalShortcut = electron.remote.globalShortcut;
 const ipcRenderer    = electron.ipcRenderer;
+const player         = PlayerActions;
```

Comment:
AppAction members are used in  the route file. Like [this](https://github.com/KeitIG/museeks/blob/master/src/js/router/routes.js#L29). How do we handle that?